src/styles/blocks/modifiers.scss

```diff
@@ -0,0 +1,18 @@
+@for $day-number from 1 through 31 {
+  .calendar__day:nth-child(#{$day-number})::before {
```

Comment:
Consider using a CSS variable for the day number instead of generating 31 classes with a preprocessor. This would reduce the size of the CSS file and make the code more maintainable.